src/components/SchemaReference.js

```diff
@@ -0,0 +1,497 @@
+      return String(t).charAt(0).toUpperCase() + String(t).slice(1);
+  }
+}
+
+function HTML({ html }) {
+  // Safe enough for our constrained formatting output
+  return <span dangerouslySetInnerHTML={{ __html: html }} />;
+}
+
+function markdownToHtml(text) {
+  if (!text) return '';
+  let html = String(text);
+
+  const escapeHtml = (s) =>
+    String(s)
+      .replace(/&/g, '&amp;')
+      .replace(/</g, '&lt;')
+      .replace(/>/g, '&gt;')
+      .replace(/"/g, '&quot;')
+      .replace(/'/g, '&#39;');
+
+  // Remove fenced code blocks delimited by triple backticks (```), with optional language
+  html = html.replace(/```[^\n]*\n[\s\S]*?```/g, '');
+
+  // Remove reference-style link definition blocks that follow a blank line
+  // e.g. "\n\n[`Builder`]: crate::Builder" or "\n\n[label]: https://..."
+  html = html.replace(/(?:\r?\n){2}(?:\s*\[[^\]]+\]:[^\n]*\r?\n?)+/g, '');
+
+  // Also remove any remaining single-line reference definitions
+  html = html.replace(/^\s*\[[^\]]+\]:\s*[^\n]+$/gm, '');
+
+  // Resolve Rust doc-style code references like [`Type`] or [`Type::member`]
+  const TYPE_MODULE_HINTS = {
+    Verify: 'settings',
+    MerkleMap: 'assertions',
+    BmffHash: 'assertions',
+    Builder: '',
+  };
+  const resolveDocsUrl = (codeRef) => {
+    let refStr = codeRef.replace(/^crate::/, '');
+
+    const parts = refStr.split('::');
+    let moduleParts = [];
+    let typeName = '';
+    let member = '';
+
+    if (parts.length === 1) {
+      // Single type name; use hints for module placement, default to crate root
+      typeName = parts[0];
+      const hinted = TYPE_MODULE_HINTS[typeName];
+      if (hinted) {
+        moduleParts = hinted ? [hinted] : [];
+      } else {
+        moduleParts = [];
+      }
+    } else if (parts.length >= 2) {
+      const last = parts[parts.length - 1];
+      const prev = parts[parts.length - 2];
+      const isLastType = /^[A-Z]/.test(last);
+      if (isLastType) {
+        typeName = last;
+        moduleParts = parts.slice(0, parts.length - 1);
+      } else {
+        member = last;
+        typeName = prev;
+        moduleParts = parts.slice(0, parts.length - 2);
+      }
+    }
+
+    if (moduleParts.length === 0 && typeName) {
+      const hinted = TYPE_MODULE_HINTS[typeName];
+      if (hinted !== undefined) moduleParts = hinted ? [hinted] : [];
+    }
+
+    const modulePath = moduleParts.length ? moduleParts.join('/') + '/' : '';
+    let url = `https://docs.rs/c2pa/latest/c2pa/${modulePath}struct.${typeName}.html`;
+
+    if (member) {
+      const anchor = /_/.test(member)
+        ? `structfield.${member}`
+        : `method.${member}`;
+      url += `#${anchor}`;
+    }
+    return url;
+  };
+
+  // Convert [`CodeRef`] to links to docs.rs, preserving code formatting
+  html = html.replace(/\[`([^`]+)`\]/g, (m, codeRef) => {
+    const url = resolveDocsUrl(codeRef);
+    return `<a href="${escapeHtml(url)}"><code>${escapeHtml(
+      codeRef,
+    )}</code></a>`;
+  });
```

Comment:
This is great that it links directly do docs.rs! Is it possible to also clear out the extra brackets following the code ref here or some other place in the code?

For example:
> [Reader::resource_to_stream](https://docs.rs/c2pa/latest/c2pa/struct.Reader.html#structfield.resource_to_stream)[crate::Reader::resource_to_stream]